### PR TITLE
tiny_slam release added

### PR DESCRIPTION
**Pull request by krinkin**

### CODE REVIEW

File: jade/distribution.yaml [hunk 1]
```diff
@@ -5744,6 +5744,11 @@ repositories:
       type: git
       url: https://github.com/OSLL/tiny-slam-ros-cpp.git
       version: master
+    release:
+      tags:
+		    release: release/jade/{package}/{version}
```
Comment (dirk-thomas) [inline jade/distribution.yaml]: Please fix the indentation for Travis to pass the tests.

Comment (krinkin) [inline jade/distribution.yaml]: done

File: jade/distribution.yaml [hunk 2]
```diff
@@ -5744,6 +5744,11 @@ repositories:
       type: git
       url: https://github.com/OSLL/tiny-slam-ros-cpp.git
       version: master
+    release:
+      tags:
+        release: release/jade/{package}/{version}
+      url: https://github.com/OSLL/tiny-slam-ros-cpp.git
```
Comment (dirk-thomas) [inline jade/distribution.yaml]: You should use a separate release repo and not the same as for the upstream.

**Comment by krinkin** [comment]
Thanks @dirk-thomas, this building releasing process is new for me

**Comment by tfoote** [comment]
@krinkin we strongly advise letting bloom open the pull request for you. It avoids many typos by entering it automatically.

**Comment by krinkin** [comment]
@tfoote 
Could you point me to doc about it. I've been used 

- http://wiki.ros.org/bloom/Tutorials/FirstTimeRelease
- http://wiki.ros.org/rosdistro/Tutorials/Indexing%20Your%20ROS%20Repository%20for%20Documentation%20Generation

Is there higher level overview of development/release-engineering process?

**Comment by tfoote** [comment]
bloom does it automatically unless you tell it not to or you haven't setup the access keys if you have 2 factor enabled.

**Pull request merged**